RPerformanceTracking.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,9 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		1696C3CA207714D700288BB5 /* RPTConfigurationTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 1696C3C9207714D700288BB5 /* RPTConfigurationTests.m */; };
+		1696C3CD2077157200288BB5 /* TestUtils.m in Sources */ = {isa = PBXBuildFile; fileRef = 1696C3CC2077157200288BB5 /* TestUtils.m */; };
+		3059D7739CD8B79D17CD17F0 /* Pods_Tests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = EDB01895B3BE708F72F5A39E /* Pods_Tests.framework */; };
 		5A8CF0CD1E9C76A1006E7544 /* UIControlTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 5A8CF0CC1E9C76A1006E7544 /* UIControlTests.m */; };
```

Comment:
We don't need commit the CocoaPods integrated version of the project file.
It's cleaner that way and easier to see real project file changes in the diff.
Please run `pod deintegrate` before committing.